test/unit/BondingManager.js

```diff
@@ -1827,10 +1847,9 @@ contract("BondingManager", accounts => {
                         [nonTranscoder, constants.NULL_ADDRESS, PERC_DIVISOR / 2, 0]
                     )
                 )
-                await fixture.roundsManager.setMockUint256(functionSig("currentRound()"), currentRound+2)
-                const endTotalActiveStake = await bondingManager.currentRoundTotalActiveStake()
-                assert.isNotOk(await bondingManager.isActiveTranscoder(nonTranscoder), "should remain inactive for the round")
-                assert.equal(startTotalActiveStake.sub(endTotalActiveStake).toNumber(), 0, "should decrease total active stake by total stake of transcoder")
+                const endBondedAmount = (await bondingManager.getDelegator(nonTranscoder))[0]
+
+                assert.equal(endBondedAmount, startBondedAmount.div(new BN(2)).toNumber(), "should decrease transcoder's bondedAmount by slashAmount")
```

Comment:
Since `bondedAmount = 0` for `nonTranscoder`, `slashTranscoder()` will not execute any of the logic within the if clause since the conditional check that `del.bondedAmount > 0`. Furthermore, `bondedAmount` does not actually decrease in this test - the assertion passes because it checks `0 == 0`. We already have a test in for `slashTranscoder()` for when the account has `bondedAmount = 0`, so here we just need to update this test so that `nonTranscoder` has `bondedAmount > 0`, but is not in the pool.

Reply:
fixed in e6d92cf